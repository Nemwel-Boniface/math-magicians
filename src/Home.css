.home {
  width: 90%;
  margin: 6% 5%;
}

.home h1 {
  font-size: 45px;
  margin-bottom: 40px;
}

.home p {
  width: 80%;
  font-size: 20px;
  margin-bottom: 20px;
}
