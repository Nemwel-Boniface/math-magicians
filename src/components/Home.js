const Home = () => (
  <div className="home w3-container w3-animate-right">
    <h1>Welcome to Nemwel&apos;s math magician page!</h1>
    <p>
      In publishing and graphic design, Lorem ipsum is a placeholder
      text commonly used to demonstrate the visual form of a document or a
      typeface without relying on meaningful content. Lorem ipsum may be used as
      a placeholder before the final copy is available
    </p>
    <p>
      In publishing and graphic design, Lorem ipsum is a placeholder text
      commonly used to demonstrate the visual form of a document or a typeface
      without relying on meaningful content. Lorem ipsum may be used as a placeholder
      before the final copy is available
    </p>
  </div>
);

export default Home;
