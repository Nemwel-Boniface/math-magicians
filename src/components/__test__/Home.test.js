import React from "react";
import { render, cleanup } from "@testing-library/react";
import Home from '../Home';

afterEach(cleanup)
test('Should create a Home snapshot', () => {
  const { asFragment } = render(<Home />)
  expect(asFragment(<Home />)).toMatchSnapshot()
})