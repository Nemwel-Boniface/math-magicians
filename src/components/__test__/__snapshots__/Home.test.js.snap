// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should create a Home snapshot 1`] = `
<DocumentFragment>
  <div
    class="home w3-container w3-animate-right"
  >
    <h1>
      Welcome to Nemwel's math magician page!
    </h1>
    <p>
      In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available
    </p>
    <p>
      In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available
    </p>
  </div>
</DocumentFragment>
`;
